zhmccli/_cmd_imageprofile.py

```diff
@@ -1531,13 +1654,50 @@ def handle_special_imageprofile_options(cmd_ctx, org_options, properties):
         properties['zaware-master-userid'] = None
     if org_options['zaware-master-password'] == '':
         properties['zaware-master-pw'] = None
+    if org_options['zaware-network-info']:
+        value = parse_yaml_flow_style(
+            cmd_ctx, '--zaware-network-info',
+            org_options['zaware-network-info'])
+        properties['zaware-network-info'] = value
+    if org_options['zaware-gateway-info'] == '':
+        properties['zaware-gateway-info'] = None
+    elif org_options['zaware-gateway-info']:
+        value = parse_yaml_flow_style(
+            cmd_ctx, '--zaware-gateway-info',
+            org_options['zaware-gateway-info'])
+        properties['zaware-gateway-info'] = value
+    if org_options['zaware-dns-info']:
+        value = parse_yaml_flow_style(
```

Comment:
Repeating

value = ...
properties[] = value

could be a helper method

Reply:
The reason  it is stored in `value` is just to avoid a line extension via backslash. The main functionality is already in a helper function, i.e. `parse_yaml_flow_style()` or `absolute_capping_value()`. Another helper function that stores the result of these functions in a dict is really overkill and would not change much.

Note that the option name is not always equal to the property name, e.g.
```
    if org_options['absolute-cp-capping'] is not None:
        value = absolute_capping_value(
            cmd_ctx, org_options, 'absolute-cp-capping')
        properties['absolute-general-purpose-capping'] = value
```

What would reduce the number of repetitions of the option names would be a change of inteerface of `parse_yaml_flow_style(cmd_ctx, option_name, value)` to `parse_yaml_flow_style(cmd_ctx, option_name, option_dict)`:
From:
```
    if org_options['ssc-dns-info']:
        value = parse_yaml_flow_style(
            cmd_ctx, '--ssc-dns-info', org_options['ssc-dns-info'])
        properties['ssc-dns-info'] = value
```
To:
```
    if org_options['ssc-dns-info']:
        value = parse_yaml_flow_style(
            cmd_ctx, 'ssc-dns-info', org_options)
        properties['ssc-dns-info'] = value
```